docs/compatibility.md

```diff
@@ -441,6 +441,42 @@ parse some variants of `NaN` and `Infinity` even when this option is disabled
 ([SPARK-38060](https://issues.apache.org/jira/browse/SPARK-38060)). The RAPIDS Accelerator behavior is consistent with
 Spark version 3.3.0 and later.
 
+### get_json_object
+
+The `GetJsonObject` operator that takes JSON input and a json path string to process. The code base
```

Comment:
This first sentence seems to be missing something -- like what it does with the input and json path string.